### ISSUE TITLE
Feature request: enable multiple document conversions by "mi6 build"

### ISSUE THREAD

**Issue by tbrowder** [skaji/mi6#117]
It would be very useful for the using module to have multiple document pod files be converted to Markdown during the **mi6 build** step.  I can think of two ways to do it:

1. Allow multiple filename entries in the **dist.ini** file under the **[ReadmeFromPod]** section, with the first entry being treated as now and other entries being converted to a file with the same basename as the input file basename but with an **.md** suffix replacing its existing suffix and the new file being placed in the top level of the repo.

2. Execute any **Makefile** ("make") existing in the top level of the repo during the **mi6 build** step.  **This would be my choice.**

**Comment by tbrowder** [comment]
I'll submit a PR.

**Comment by tbrowder** [comment]
I have made an attempt at this on a PR. It is failing the xt/01* test I made for it.
I will submit the draft PR if you are interested. Note I have not yet touched the documentation, and I have inserted debugging code and comments that will be cleaned up before any formal submission. And I will also update the documentation for the PR.

I have been running the tests in the following manner:

    $ raku -I. xt/01*

My current approach is to allow a second file (filename2) in the ReadmeFromPod dist.ini section. But I am obviously missing something in the application.
